### ISSUE TITLE
IntelliJ IDEA: kotlin-dsl plugin does not configure action receiver in buildSrc

### ISSUE THREAD

**Issue by ingokegel** [gradle/kotlin-dsl#413]
To reproduce, open 

    samples/multi-kotlin-project-with-buildSrc/buildSrc/src/main/kotlin/utils.kt

and add

    copy {
        from("source")
        into("target")
    }

into the `kotlinProject` method. The IDE shows this as red code: 

<img width="391" alt="screen shot 2017-07-09 at 23 32 15" src="https://user-images.githubusercontent.com/1199475/27997729-fed9fa94-64fe-11e7-8d29-862c485a72f6.png">

but gradle compiles it correctly on the command line.

Adding `it` fixes this for the IDE:

<img width="390" alt="screen shot 2017-07-09 at 23 32 35" src="https://user-images.githubusercontent.com/1199475/27997737-2788b1ec-64ff-11e7-93ff-0bfab13566cb.png">

but then gradle fails on the command line.

**Comment by eskatos** [comment]
Hi @ingokegel,
Thanks for the report.
Which version of IntelliJ and it's Kotlin plugin are you using?
The IntelliJ Kotlin plugin >=1.1.3 is required for this to work.

**Comment by ingokegel** [comment]
I'm using Kotlin 1.1.3-2 and IJ 2017.1.5, same problem with IJ 2017.2-RC2, reproduced on two separate machines (Windows/macOS)

**Comment by bamboo** [comment]
Thanks for the report, @ingokegel. We're looking into it.

**Comment by bamboo** [comment]
Blocked by #414.

**Comment by bamboo** [comment]
The `kotlin-dsl` plugin is now doing the right thing and this should work once a new IntelliJ Kotlin plugin is out with the fix for https://youtrack.jetbrains.com/issue/KT-18062

**Comment by eskatos** [comment]
https://youtrack.jetbrains.com/issue/KT-18062 did fix it for production code, e.g. Gradle plugins written in Kotlin, but not for `buildSrc`.

![image](https://user-images.githubusercontent.com/132773/29972109-3c176ddc-8f2b-11e7-84ac-c9d457a8e423.png)

**Comment by eskatos** [comment]
Blocked by #476

**Comment by eskatos** [comment]
Closing as duplicate of #476
